src/Umbraco.Web/PropertyEditors/DatePreValueEditor.cs

```diff
@@ -6,5 +6,23 @@ internal class DatePreValueEditor : PreValueEditor
     {
         [PreValueField("format", "Date format", "textstring", Description = "If left empty then the format is YYYY-MM-DD. (see momentjs.com for supported formats)")]
         public string DefaultValue { get; set; }
+
+        [PreValueField("showToday", "Show today", "boolean", Description = "Choose if today should be shown.")]
+        public bool ShowToday { get; set; }
```

Comment:
Maybe we just want the today label always to be shown (the small triangle). I think there were some issues using bool instead of string, where it was undefined in the controller when using bool instead of string.